src/logging.h

```diff
@@ -199,11 +200,14 @@ static inline void LogPrintf_(const std::string& logging_function, const std::st
     }
 }
 
-
 #define LogPrintLevel_(category, level, ...) LogPrintf_(__func__, __FILE__, __LINE__, category, level, __VA_ARGS__)
 
+// Log unconditionally.
 #define LogPrintf(...) LogPrintLevel_(BCLog::LogFlags::NONE, BCLog::Level::None, __VA_ARGS__)
 
+// Log unconditionally, prefixing the output with the passed category name.
+#define LogPrintfCategory(category, ...) LogPrintLevel_(category, BCLog::Level::None, __VA_ARGS__)
```

Comment:
It's `LogPrintLevel` and `LogPrintfCategory` that's a bit inconsistent. I think we can leave out the `f` here.